cpu/cortexm_common/thread_arch.c

```diff
@@ -405,6 +408,7 @@ void __attribute__((naked)) __attribute__((used)) isr_pendsv(void) {
     "mov    r1, sp                    \n" /* restore the user mode SP */
     "msr    psp, r1                   \n" /* for this write it to the PSP reg */
     "mov    sp, r12                    \n" /* and get the parked MSR SP back */
+    "ibs                              \n" /* flush pipeline */
```

Comment:
here I'm not sure. the write to sp doesn't need this, the msr instruction above might. but as the psp is only used after exception return, I don't think it is really needed.